babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -340,6 +340,60 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testTableAttributeDataBlockSizeMinimum() {
+    final String sql = "create table foo, minimum datablocksize (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, MINIMUM DATABLOCKSIZE (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeDataBlockSizeMin() {
+    final String sql = "create table foo, min datablocksize (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, MINIMUM DATABLOCKSIZE (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeDataBlockSizeMaximum() {
+    final String sql = "create table foo, maximum datablocksize (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, MAXIMUM DATABLOCKSIZE (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeDataBlockSizeMax() {
+    final String sql = "create table foo, max datablocksize (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, MAXIMUM DATABLOCKSIZE (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeDataBlockSizeDefault() {
+    final String sql = "create table foo, default datablocksize (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, DEFAULT DATABLOCKSIZE (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeDataBlockSizeValueNoUnit() {
+    final String sql = "create table foo, datablocksize = 12123 (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, DATABLOCKSIZE = 12123 BYTES (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeDataBlockSizeValueBytes() {
+    final String sql = "create table foo, datablocksize = 12123 bytes (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, DATABLOCKSIZE = 12123 BYTES (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeDataBlockSizeValueKbytes() {
+    final String sql = "create table foo, datablocksize = 42.123 kbytes (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, DATABLOCKSIZE = 42.123 KILOBYTES (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeDataBlockSizeValueKilobytes() {
+    final String sql = "create table foo, datablocksize = 2e4 kilobytes (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, DATABLOCKSIZE = 2E4 KILOBYTES (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
```

Comment:
Could also add a test case checking that having both a keyword before DATABLOCKSIZE and an assignment after (ex "MAX DATABLOCKSIZE = 42" ) fails.

Reply:
+1

Reply:
Good point, I'll add a test case for that